cmd/dns/cmd.go

```diff
@@ -31,7 +24,7 @@ func getDefaultRegion() string {
 	}
 }
 func NewCmdValidateDns() *cobra.Command {
-	config := dnsConfig{}
+	config := cloudclient.CmdOptions{}
```

Comment:
Why was this changed? The DNS test doesn't need all the same cmd options as the egress test. Each test may have its  own subset or superset of options it needs that aren't related to the cloudclient